From ff36b01d0954e56acb3c63625095fca7a2d5923c Mon Sep 17 00:00:00 2001
From: Amir Vadai <avadai@redhat.com>
Date: Thu, 11 Sep 2014 12:01:49 -0400
Subject: [ethernet] mlx4: mlx4_en_[gs]et_priv_flags() can be static

Message-id: <8dc4db9cf774cf40401c6b78981066dd4e67cc76.1410425016.git.avadai@redhat.com>
Patchwork-id: 93711
O-Subject: [Patch RHEL7.1 V1 173/197] net/mlx4_en: mlx4_en_[gs]et_priv_flags() can be static
Bugzilla: 1107617 1107618 1107619 1107620
RH-Acked-by: Jes Sorensen <Jes.Sorensen@redhat.com>

From: Fengguang Wu <fengguang.wu@intel.com>

Fixes sparse warning intrduced by commit 0fef9d0 ("net/mlx4_en: Disable
blueflame using ethtool private flags")

Signed-off-by: Fengguang Wu <fengguang.wu@intel.com>
Signed-off-by: Amir Vadai <amirv@mellanox.com>
Signed-off-by: David S. Miller <davem@davemloft.net>
(cherry picked from commit 3f6148e76a58eceef1435b65afecaf448b509cfd)
Signed-off-by: Amir Vadai <avadai@redhat.com>
---

Signed-off-by: Jarod Wilson <jarod@redhat.com>

diff --git a/drivers/net/ethernet/mellanox/mlx4/en_ethtool.c b/drivers/net/ethernet/mellanox/mlx4/en_ethtool.c
index a96b01a..245f3b7 100644
--- a/drivers/net/ethernet/mellanox/mlx4/en_ethtool.c
+++ b/drivers/net/ethernet/mellanox/mlx4/en_ethtool.c
@@ -1221,7 +1221,7 @@ static int mlx4_en_get_ts_info(struct net_device *dev,
  return ret;
 }
 
-int mlx4_en_set_priv_flags(struct net_device *dev, u32 flags)
+static int mlx4_en_set_priv_flags(struct net_device *dev, u32 flags)
 {
  struct mlx4_en_priv *priv = netdev_priv(dev);
  bool bf_enabled_new = !!(flags & MLX4_EN_PRIV_FLAGS_BLUEFLAME);
@@ -1256,7 +1256,7 @@ int mlx4_en_set_priv_flags(struct net_device *dev, u32 flags)
  return 0;
 }
 
-u32 mlx4_en_get_priv_flags(struct net_device *dev)
+static u32 mlx4_en_get_priv_flags(struct net_device *dev)
 {
  struct mlx4_en_priv *priv = netdev_priv(dev);
 
-- 
1.7.1